### PR TITLE
admin home

### PR DESCRIPTION
**Pull request by namnn17b3**
## Related Tickets
https://edu-redmine.sun-asterisk.vn/issues/74055

## WHAT (optional)
- Change number items `completed/total` in admin page.

## HOW
- I edit js file, inject not_vary_normal items in calculate function.

## WHY (optional)
- Because in previous version - number just depends on `normal` items. But in new version, we have `state` and `confirm_state` depends on both `normal` + `not_normal` items.

## Evidence (Screenshot or Video)
image
image

### CODE REVIEW

**Comment by namnn17b3** [comment]
ready

**Comment by namnn17b3** [comment]
ready

File: src/views/admin/admin-home.ejs [hunk 1]
```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Teacher Dashboard</title>
```
Comment (huongnt-2545) [inline src/views/admin/admin-home.ejs]: I18n text

File: src/public/js/admin.home.js [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+const endDateInput = document.querySelector('#endDate');
+const btnDrawChart = document.querySelector('.btn-draw-chart');
+
+const ADMIN_STATISTIC_MOVIE_TIME_LIMIT = 30;
+
+endDateInput.max = new Date().toISOString().split('T')[0];
+endDateInput.value = endDateInput.max;
+startDateInput.value = new Date(
+  new Date(endDateInput.value).getTime()
+  - ADMIN_STATISTIC_MOVIE_TIME_LIMIT * 24 * 60 * 60 * 1000
```
Comment (huongnt-2545) [inline src/public/js/admin.home.js]: `ADMIN_STATISTIC_MOVIE_TIME_LIMIT * 24 * 60 * 60 * 1000` -> set giá trị vào hằng số ở trên luôn e

File: src/services/admin/admin.home.service.ts [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+  }
+
+  public async getAdminRevenueInTimeRange(timeRange: any) {
+    const { startDate, endDate } = timeRange;
+    const sq = this.ticketRepository
+      .createQueryBuilder('ticket')
+      .select('distinct bill.id as bill_id')
+      .addSelect('bill.pay_time')
+      .addSelect('bill.total_price_from_ticket')
+      .addSelect('bill.total_price_from_food')
```
Comment (huongnt-2545) [inline src/services/admin/admin.home.service.ts]:
```suggestion
      .addSelect(['bill.pay_time', 'bill.total_price_from_ticket', ...])
```

**Comment by namnn17b3** [comment]
ready

**Review by huongnt-2545 — APPROVED** [review]

**Pull request merged**